foodgroup/admin.go

```diff
@@ -68,16 +110,21 @@ func (s AdminService) InfoQuery(_ context.Context, sess *state.Session, frame wi
 		}
 	}
 
-	// wire.AdminTLVRegistrationStatus is used in the AIM Preferences > Privacy panel to control
-	// Allow users who know my e-mail address to find...
-	//	o Nothing about me - wire.AdminInfoRegStatusNoDisclosure
-	//	o Only that I have an account - wire.AdminInfoRegStatusLimitDisclosure
-	//	o My screen name - wire.AdminInfoRegStatusFullDisclosure
 	if _, hasRegStatus := body.TLVRestBlock.Slice(wire.AdminTLVRegistrationStatus); hasRegStatus {
```

Comment:
Is it possible that all 3 of these items be requested in the same request?

Reply:
For the InfoQuery it seems possible. I can update so it handles one or more in the same request.

For InfoChangeRequest it seems possible, although it's unclear how error handling would work in that case - if you had 2 items fail, how do you send the proper error back for both? And now that I'm rereading the docs, I might be using the wrong snac for errors - I'm using AdminErr but it seems to also work if I set it in an InfoChangeReply snac.

I'll do some more testing on this.

Reply:
After testing and walking through libpurple

- We should only expect to receive a request for 1 item per snac. We can easily change this later if a client pops up that expects more than 1 item per snac for some reason.
- We should actually be sending errors back in AdminInfoReply or AdminInfoChangeReply snacs, not AdminErr. As I have it now, AdminErr works on AIM clients but pidgin specifically looks for those snac subgroup types to know if the change was a success or not.

So I'm going to leave what I have as-is but update the error responses to not use AdminErr.